File: CS/WindowsApplication3/MVVM/Data/Customer.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;

namespace MVVMSample {
    public class Customer {
        [Required]
        public int ID { get; set; }
        [Required]
        public string FirstName { get; set; }
        [Required]
        public string LastName { get; set; }
    }
}

File: VB/WindowsApplication3/MVVM/Data/Customer.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.Linq
Imports System.Text

Namespace MVVMSample
    Public Class Customer
        <Required> _
        Public Property ID() As Integer
        <Required> _
        Public Property FirstName() As String
        <Required> _
        Public Property LastName() As String
    End Class
End Namespace
